src/components/AnimatedFlatList.js

```diff
@@ -0,0 +1,11 @@
+import * as React from 'react';
+import { FlatList } from 'react-native';
+import createAnimatedComponent from '../createAnimatedComponent';
+
+const FlatListWithEventThrottle = React.forwardRef((props, ref) => (
+    <FlatList scrollEventThrottle={0.0001} {...props} ref={ref} />
```

Comment:
Why do you set scrollEventThrottle to such value? Why exactly that value?

Reply:
I can now see your comment under an issue. I am hesitant about your proposal to set `scrollEventThrottle` to `0.0001` value. As stated above I am not sure why such a value.
CC @jakub-gonet

Reply:
For me, it looks like a hack covering the underlying issue.